src/support/timestamp.c

```diff
@@ -116,7 +116,7 @@ __wt_verbose_timestamp(WT_SESSION_IMPL *session, wt_timestamp_t ts, const char *
     char ts_string[WT_TS_INT_STRING_SIZE];
 
     __wt_verbose(
-      session, WT_VERB_TIMESTAMP, "Timestamp %s: %s", __wt_timestamp_to_string(ts, ts_string), msg);
```

Comment:
This could possibly effect MongoDB logging and a other tests? I don't think we should make this change on a whim.